plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/help-scm.html

```diff
@@ -1,7 +1,7 @@
 <div>
     Specify where to obtain a source code repository containing your Groovy script.
-    It will be checked out somewhere on the Jenkins master and used to load your Pipeline script.
+    It will be checked out somewhere on the Jenkins controller and used to load your Pipeline script.
     (If you wish to use other files from the same repository during your Pipeline,
-    you will need to check them out separately on some slave;
+    you will need to check them out separately on some node;
```

Comment:
should be Node.

```suggestion
    you will need to check them out separately on some agent;
```

Reply:
here I thought you need to do the separate checkout even if your workspace is in the built-in node on the controller machine 🤷 But that's a discouraged usecase anyway.